sql/telemetry_derived/asn_aggregates_v1/query.sql

```diff
@@ -105,6 +105,13 @@ events_with_asn AS (
     asn_ip_address_range
   USING
     (network_ip, mask)
+  GROUP BY
+    submission_date,
+    client_id,
+    canary,
+    event_category,
+    event_object,
+    autonomous_system_number
```

Comment:
Grouping them here should ensure that for each client only up to one doh event will be aggregated in the final part of the query.

Reply:
This way `n_clients` would still refer to the total number of distinct client IDs part of a specific ASN

Reply:
It's a little more concise to write SELECT DISTINCT at line 86.